imports/plugins/core/accounts/client/containers/addressBookContainer.js

```diff
@@ -0,0 +1,214 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { compose } from "recompose";
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { i18next } from "/client/api";
+import { Countries } from "/client/collections";
+import * as Collections from "/lib/collections";
+import { registerComponent, composeWithTracker } from "@reactioncommerce/reaction-components";
+import AddressBook from "../components/addressBook";
+
+/**
+ * @method updateAddress
+ * @summary helper function that updates an address in the account's addressBook via a meteor method.
+ * @since 2.0.0
+ * @param {Object} address - address to be updated.
+ * @param{String} property - property to be updated.
+ * @return {Promise}
+ */
+function updateAddress(address, property) {
+  return new Promise((resolve, reject) => {
+    if (property) {
+      Meteor.call("accounts/addressBookUpdate", address, null, property, (error, result) => {
+        if (error || !result) reject(i18next.t("addressBookGrid.somethingWentWrong", { err: error.message }));
+        if (result) resolve(result);
+      });
+    } else {
```

Comment:
I'm not entirely sure that the `if (property)` check and this `else` are necessary. If `property` is falsy, then it will just send method arguments `address, null, null`, which will be treated the same way, looking at the server method code.